packages/components/src/color-picker/inputs.js

```diff
@@ -230,8 +230,8 @@ export class Inputs extends Component {
 		} else if ( this.state.view === 'hsl' ) {
 			return (
 				<fieldset>
-					<VisuallyHidden as="legend">
-						{ __( 'Color value in HSL' ) }
+					<VisuallyHidden>
+						<legend>{ __( 'Color value in HSL' ) }</legend>
 					</VisuallyHidden>
```

Comment:
This change and the one above will break form semantics; best leave things as they were.